Week1/cash.c

```diff
@@ -0,0 +1,25 @@
+#include <cs50.h>
+#include <stdio.h>
+
+void cashback();
+
+int main(void){
+    float owe = get_float("Change owed: ");
+    int penny = (int)(owe*100);
```

Comment:
Problems here: implicit floor cast; variable names.

```suggestion
    int owed_change_in_penny = (int) (owed_change * 100.0);
```